docs/api/synchronization.rst

```diff
@@ -0,0 +1,160 @@
+.. _api-synchronization:
+
+Synchronization
+###############
+
+This section describes the basic aspects of synchronization using *Kinto*.
+
+.. note::
+
+    If you are looking for a ready-to-use synchronization solution,
+    jump to :ref:`sync-implementations`.
+
+
+The main idea consists in maintaining a local database up-to-date
+with the *Kinto* server. Remote changes are downloaded and applied
+on the local data. Local changes are uploaded using HTTP headers to control
+concurrency and overwrite.
+
+In short:
+
+#. Poll for remote changes using ``?_since=<timestamp>``
+#. Apply changes locally
+#. Post creations
+#. Use concurrency control to send updates and deletes
+
+
+Polling for changes
+===================
+
+When fetching the collection records with ``?_since=<timestamp>``, *Kinto* returns
+every records that were created/updated/deleted **after** this timestamp.
+
+#. Cold sync → no ``?_since``.
+#. ``GET /buckets/default/collections/articles?_since=<timestamp>`` (with header ``If-Unmodified-Since = <timestamp>``)
+#. If response is ``304 Not modified``, done → up to date, nothing to do.
+#. If response is ``200 OK`` store ``ETag`` response header value for next
+   synchronization and handle the obtained list of changes.
+
+.. note::
+
+    * Deleted records have an attribute ``delete: true``
+    * Created/updated records are both returned in their new version
+    * Since *Kinto* does not keep any history, there is no *diff* for updates
+
+
+Paginated changes
+-----------------
+
+By default, *Kinto* does not paginate the records list. If an explicit limit is
+set in the server settings or using the ``?_limit=<nb>`` parameter, then polling for
+changes will be paginated.
+
+It basically consists in fetching the list until the ``Next-Page`` is present.
+
+Since changes can occur between the first and the last page, the synchronization
+process is a bit more complex.
+
+#. Cold sync → no ``?_since``.
+#. ``GET /buckets/default/collections/articles?_since=<timestamp>`` (with header ``If-Unmodified-Since = <timestamp>``)
+#. If response is ``304 Not modified``, done → up to date, nothing to do.
```

Comment:
As with the list above, don't be afraid to be a little more verbose with the contents of this list, too. :smile: